index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+  <head>
+    
+    <meta charset="utf-8">
+    
+    <meta name="description" content="Beyonce runs the world">
+    
+    <meta name="keywords" content="beyonce, formation, lemonade">
+    
+    <title>Beyonce | Formation Bio</title>
+    
+    <link rel="stylesheet" href="css/styleindex.css">
+  
+  </head>
+
+  <body>
+
+    <img class="logo" src="images/beyonce_logo.png" alt="logo">
+
+    <hr>
+
+    <nav>
+      <a href="index.html"><strong>Bio</strong></a>
+      |
+      <a href="experience.html"><strong>Work</strong></a>
+    </nav>
+
+    <hr>
+    
+    <div class="bio_content">
+      
+      <img class="bio_image" src="images/bey_home.jpg" alt="Growing Up">
```

Comment:
@mbhat25 look to see if you can find a pattern in your markup that suits you design. Meaning could you use a div or any other container to adjust your markup.